### PR TITLE
Fix a few issues in the setup and install process

### PR DESCRIPTION
**Pull request by willpoorman**
2611b4b: Fix check for ffmpeg during setup script
Yeah, not sure how I ever though `type blah` was going to check if `ffmpeg` was installed. This fixes it to actually check for `ffmpeg`. Since it takes a long time to pull it from source and compile it. This will cut down on setup time if you already have it installed.

7ab3993: Remove unneeded depedency
This dependency causes occassional `pip install -r requirements.txt` to fail. It's not needed and is technically a bug: https://stackoverflow.com/questions/39577984/what-is-pkg-resources-0-0-0-in-output-of-pip-freeze-command

1ea49b2: Add install of package needed for python ctypes
This is generally already installed by the time you instally python for the first time. But somehow I missed getting it installed on my Windows Subsystem for Linux setup. So I'm including it here to be safe.

18842dc: Add vscode settings
Allows consistent experience for anyone developing this using vscode

This one isn't  really related to anything else in the PR but it's such a small thing I figured I'd just tack it on

b63d28d Add chromedriver install to setup script

7b19a78 Add pylama and depedencies to requirements.txt
Choosing pylama as default linter because it wraps pylint, flake8 and several other useful linting and code analysis tools
e06380a: Update vscode settings to enable pylama linter

81a6c4d: Add unzip to install list
I don't believe it comes installed by default, so let's make sure it's there

### CODE REVIEW

**Comment by willpoorman** [comment]
Related question, @michaelhillebrand , any reason we build `ffmpeg` from source rather than installing the prebuilt package? Looks like it's got prebuilt packages for most OS's: http://ffmpeg.org/download.html

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "python.pythonPath": "venv/bin/python",
+    "python.linting.pylintEnabled": false,
+    "python.linting.pylamaEnabled": true,
+    "python.linting.enabled": true
+}
```
Comment (aleksandrsvoboda) [inline .vscode/settings.json:6]: Probably need a new line at the end of the file.

Comment (willpoorman) [reply .vscode/settings.json:6]: it was an auto generated file, so it didn't get one automatically. To save you guys having to reapprove this, I'll just add the newline as an addon to some later PR

**Comment by willpoorman** [comment]
[Update setup.sh to download and place chrome driver for Selenium](https://trello.com/c/Y13FhSDO/23-update-setupsh-to-download-and-place-chrome-driver-for-selenium)

**Pull request merged**